src/google/Scatter.js

```diff
@@ -0,0 +1,215 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./CommonND", "../common/HTMLWidget"], factory);
+    } else {
+        root.Scatter = factory(root.d3, root.google_CommonND, root.common_HTMLWidget);
+    }
+}(this, function (d3, CommonND, HTMLWidget) {
+
+    function Scatter() {
+        CommonND.call(this);
+        this._class = "google_Scatter";
+        this._tag = "div";
+        this._chartType = "ScatterChart";
+    }
+    Scatter.prototype = Object.create(CommonND.prototype);
+
+    /**
+     * Publish Params Unique To This Widget
+     */
+    Scatter.prototype.publish("aggregationTarget", "auto", "string", "How multiple data selections are rolled up into tooltips: 'category'- Group selected data by x-value; 'series'- Group selected data by series; 'auto'- Group selected data by x-value if all selections have the same x-value, and by series otherwise; 'none'- Show only one tooltip per selection.  aggregationTarget will often be used in tandem with selectionMode and tooltip.trigger",null,{tags:['Basic']});
+    Scatter.prototype.publish("curveType", "none", "set", "Controls the curve of the lines when the line width is not zero. Can be one of the following: 'none' - Straight lines without curve; 'function' - The angles of the line will be smoothed..",['none', 'function'],{tags:['Basic']});
+    Scatter.prototype.publish("pointShape", "circle", "set", "The shape of individual data elements: 'circle', 'triangle', 'square', 'diamond', 'star', or 'polygon'.",['circle', 'triangle', 'square', 'diamond', 'star', 'polygon'],{tags:['Basic']});
+    Scatter.prototype.publish("pointSize", 7, "number", "Diameter of data points, in pixels. Use zero to hide all points.",null,{tags:['Basic']});
+    Scatter.prototype.publish("pointsVisible", true, "boolean", "Determines whether points will be displayed. Set to false to hide all points.",null,{tags:['Basic']});
+    Scatter.prototype.publish("selectionMode", 'single', "set", "When selectionMode is 'multiple', users may select multiple data points.",['single','multiple'],{tags:['Basic']});
+
+     /**
+     * Publish Params Common To Other Libraries
+     */
```

Comment:
group and single space params where you can takes up less space and easier to read? easier for me to read...